tests/sceneManager_test.cpp

```diff
@@ -11,6 +11,68 @@ using objects::Object;
 using objects::TriangleObj;
 
 const double kSkipFreq = 1000;
+const core::Vec3 kSkipVec3(0, 0, 0);
 generators::FakeRandomGen fakeRand;
 
-// TODO: Change all doubles to float
+class SceneManagerTest : public ::testing::Test
+{
+protected:
+    SceneManager manager;
+    std::vector<objects::EnergyCollector *> energyCollectors;
```

Comment:
No need to cache this here. Just call getEnergyCollectors() in performHitAtEnergyCollectors

Reply:
ok